### PR TITLE
KFSPTS-18056 fixing concur event notification integration

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

File: src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        cuMarshalService = null;
+        xmlFile = null;
+    }
+    
+    @Test
+    public void marshalEventNoticationListFile() throws JAXBException {
```
Comment (ajd299) [inline src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java]: Typo: Notication => Notification

File: src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import edu.cornell.kfs.sys.service.CUMarshalService;
+import edu.cornell.kfs.sys.service.impl.CUMarshalServiceImpl;
+
+public class ConcurEventNotificationDTOTest {
+    private static final String EXAMPLE_FILE_NAME = "src/test/resources/edu/cornell/kfs/concur/rest/xmlObjects/fixture/event-notification-payload-example.xml";
+    private static final String EQUAL_ASSERT_STATEMENT = "Expected value should equal the actual value.";
```
Comment (ajd299) [inline src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java]: I do not think this is used? Do we need to keep it?

File: src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    public void setUp() throws Exception {
+        cuMarshalService = new CUMarshalServiceImpl();
+        xmlFile = new File(EXAMPLE_FILE_NAME);
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        cuMarshalService = null;
+        xmlFile = null;
+    }
```
Comment (ajd299) [inline src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java]: Is the "throws Exception" needed here?

File: src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java (hunk 1 above)
Comment (jhulslander) [reply src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java]: will fix

File: src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java (hunk 2 above)
Comment (jhulslander) [reply src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java]: removed

File: src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java (hunk 3 above)
Comment (jhulslander) [reply src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java]: nope, it is not needed, I shall remove

**Comment by jhulslander** [comment]
I was going to leave failed-event-queue-response-example.xml in, but there is no reason not to take it out ..... I shall attach it to the follow up user story to put the unit test back in

**Comment by jhulslander** [comment]
I've made some changes this is ready for review again

File: src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    private static final String EXAMPLE_FILE_NAME = "src/test/resources/edu/cornell/kfs/concur/rest/xmlObjects/fixture/event-notification-payload-example.xml";
+    
+    private CUMarshalService cuMarshalService;
+    private File xmlFile;
+    
+    @Before
+    public void setUp() throws Exception {
+        cuMarshalService = new CUMarshalServiceImpl();
+        xmlFile = new File(EXAMPLE_FILE_NAME);
+    }
```
Comment (ajd299) [inline src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java]: Is the throws Exception needed here?

File: src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+    @After
+    public void tearDown(){
```
Comment (ajd299) [inline src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java]: minor format fix, there is currently no space between the end parenthesis and opening curly brace.

File: src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java (hunk 4 above)
Comment (jhulslander) [reply src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java]: nope., I shall remove

File: src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java (hunk 5 above)
Comment (jhulslander) [reply src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java]: fixed

**Comment by jhulslander** [comment]
changed made, this is ready for review again

**Pull request merged**